port/raspberrypi/rp2xxx/src/hal/i2c.zig

```diff
@@ -195,6 +195,7 @@ test "i2c.translate_baudrate" {
     try std.testing.expectError(ConfigError.InputFreqTooLow, translate_baudrate(1_000_000, 31_900_000));
 }
 
+// TODO: Why create this empty struct?
```

Comment:
I will remove this, but I am wondering why we do this. My hal does not. I don't think it's necessary?